docs/source/system_reference_guide/ade_custom_extensions.rst

```diff
@@ -0,0 +1,10 @@
+ADE Custom Extensions
+=======================================
+
```

Comment:
There's no text explaining what is in this section.

Reply:
True!  There is also no explanation on the existing page: https://docs.maap-project.org/en/develop/ade_custom_extensions.html

Reply:
(I am not opposed to adding a description as part of the re-org)

Reply:
Let just open a ticket to go through and add Descriptions to all the landing pages, and not do it in this PR.

Reply:
see: https://github.com/MAAP-Project/maap-documentation/issues/168